### PR TITLE
Stronger security biosuits

### PR DESCRIPTION
**Pull request by ghost**
To make the security biosuit more consistent with how other security exosuits tend to work, that is the assumption that they have some armour rather than none. Also a slot for your taser.

Adds some armour to security biosuits to be a bit weaker than security winter coats.
Adds the option to put regular security items on the suit.

[consistency]

:cl:
 * rscadd: added armour and more items can go on the biosuit

### CODE REVIEW

**Comment by ghost** [comment]
This is my first time using github tear me apart.

**Comment by 9percents** [comment]
can you add a small line of description that indicates that it is armoured?
otherwise, thanks

File: code/modules/clothing/suits/bio.dm [hunk 1]
```diff
@@ -48,9 +48,27 @@
 //Security biosuit, grey with red stripe across the chest
 /obj/item/clothing/head/bio_hood/security
 	icon_state = "bio_security"
+	armor = list(melee = 25, bullet = 10, laser = 15, energy = 5, bomb = 5, bio = 100, rad = 20)
 
 /obj/item/clothing/suit/bio_suit/security
 	icon_state = "bio_security"
+	 allowed = list(
```
Comment (jknpj) [inline code/modules/clothing/suits/bio.dm]: indent

Comment (ghost) [reply code/modules/clothing/suits/bio.dm]: Thanks for pointing it out.

**Comment by ghost** [comment]
> can you add a small line of description that indicates that it is armoured?
> otherwise, thanks

Good point.

File: code/modules/clothing/suits/bio.dm [hunk 2]
```diff
@@ -47,10 +47,32 @@
...
 	icon_state = "bio_security"
+	allowed = list(
+		/obj/item/weapon/gun/energy,
+		/obj/item/weapon/reagent_containers/spray/pepper,
+		/obj/item/weapon/gun/projectile,
+		/obj/item/ammo_storage,
+		/obj/item/ammo_casing,
+		/obj/item/weapon/melee/baton,
+		/obj/item/weapon/handcuffs,
+		/obj/item/weapon/gun/lawgiver,
```
Comment (jknpj) [inline code/modules/clothing/suits/bio.dm:67]: why not just any /obj/item/weapon/gun?

Comment (ghost) [reply code/modules/clothing/suits/bio.dm:67]: Because I was just copying the list from /obj/item/clothing/suit/armor for consistency's sake.

Comment (jknpj) [reply code/modules/clothing/suits/bio.dm:67]: Yes, copypaste makes our code more consistent and easier to maintain.

Comment (ghost) [reply code/modules/clothing/suits/bio.dm:67]: Clearly someone thought that security armour should not use weapon/gun and they may have had good reason to do so. This is not the PR to change that. If you'd rather this PR be titled "a complete overhaul of our exosuits" then that is on you.

**Pull request merged**